routes/user/user.js

```diff
@@ -87,6 +87,7 @@ exports.getUser = function(req, res, next) {
 exports.createUser = function(req, res, next) {
     var user = new User(req.body);
     user.us_activation_token = common.getRandomToken();
+    user.us_password = sha1(user.us_password);
```

Comment:
Incorrect place to be hashing.  Please remove and do hashing on frontend application